docs/src/core/transaction-request/MERCHANT_INTEGRATION.md

```diff
@@ -102,19 +102,21 @@ const post = async (request, response) => {
     // create spl transfer instruction
     const splTransferIx = await createSplTransferIx(sender, connection);
 
-    // create the transaction
-    const transaction = new Transaction();
-
-    // add the instruction to the transaction
-    transaction.add(splTransferIx);
+    const { blockhash } = await connection.getLatestBlockhash();
```

Comment:
I might propose using the empty blockhash (`111...`) here as described in the spec. The wallet is supposed to fill this in if empty, which is better than providing one. Also getting a blockhash delays the response.